concepts/submission_date.md

```diff
@@ -0,0 +1,65 @@
+# Submission Date
+
+This document will help you understand `submission_date`, why are standardizing on it, and how it
+differs from `activity_date`.
+
+## Table of Contents
+
+<!-- toc -->
+
+# What is Submission Date
```

Comment:
Please describe here what `activity_date` is as well, which is based on `subsession_start_date`.

Reply:
fixed